hw/bitstream/README.md

```diff
@@ -0,0 +1,129 @@
+# FPGA Bitstreams
+
+## Overview
+
+The build infrastructure supports the ability to override the ROM and OTP
+default values in precompiled FPGA bitstreams. The following configurations
+are available for use in FPGA based test programs:
+
+1. `rom_with_<key_type>_keys_<life_cycle_state>`
+3. `test_rom`
```

Comment:
I'm not sure about the indexing here :-) What happened to 2?!

Reply:
Good catch, I don't know where it went! Fixed.